src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.exception.InvalidValueExeption;
+import core.basesyntax.service.CsvFileReaderService;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileReaderServiceImplTest {
+    private static CsvFileReaderService fileReaderService;
+    private static final String VALID_FILE_PATH = "src/test/resources/TestData.csv";
+    private static final String INVALID_FILE_PATH = "src/resources/NonexistentFile.csv";

```

Comment:
```suggestion
    private static final String INVALID_FILE_PATH = "src/test/resources/NonexistentFile.csv";

```

Reply:
But I'm specificslly passing an invalid path to make my test work